### PR TITLE
Issue 7

### LINKED ISSUE #7: Classes lacking documentation
Classes defined in inflammation/models.py are lacking documentation/docstrings.

### PR DESCRIPTION
**Pull request by h4pZ**
Updated the requirements missing for `views.py` as well as the docstrings and typehints

### CODE REVIEW

File: inflammation/views.py [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
+import numpy as np
 from matplotlib import pyplot as plt
+from inflammation.models import Patient
 
 
-def visualize(data_dict):
+def visualize(data_dict: Dict[str, np.ndarray]):
     """Display plots of basic statistical properties of the inflammation data.
 
     :param data_dict: Dictionary of name -> data to plot
```
Comment (harishravip) [inline inflammation/views.py:11]: @isabelle-gaffney @carlosroca-az @h4pZ Do we want to add a type hint at the parameter def as well?

Comment (h4pZ) [reply inflammation/views.py:11]: It is nice, that way your type checker and LSP can help when coding

Comment (isabelle-gaffney) [reply inflammation/views.py:11]: I personally really like these, and hopefully not too much work to add them throughout, but happy to be disagreed with!

**Comment by isabelle-gaffney** [comment]
LGTM

File: inflammation/views.py (hunk 1 above)
Comment (ghost) [reply inflammation/views.py:11]: It is nice to have, if time allows

Comment (h4pZ) [reply inflammation/views.py:11]: ditto

**Pull request merged**